README.md

```diff
@@ -223,6 +234,47 @@ DISTRO_EXTRA_MIDDLEWARES:
   - middleware.test.2
 ```
 
+## How to add extra files requirements
+
+You should set the variable **INSTALL_EXTRA_FILE_REQUIREMENTS** in your config.yml file if you need to install extra files with. The structure should be like:
```

Comment:
The first time I read this, I didn't understand why I needed this. I read where you use it but I am still unsure how to use it. This is related to the command `tutor distro enable-private-packages`? Do I need this configuration to use that command?
It probably is a good idea to explain what is an extra file requirement, how the content looks, and if we need something extra; for example, if I decide to add one random file to a file directory in that, I think I need to have that file in a specific place.
I feel this part needs more info because it is not so easy to understand why I need this and what I need to do in order to do this work.

Reply:
It is just to install requirements files like: base.txt, coverage.txt, development.txt, doc.txt, testing.txt, ...
It depends on what the user needs at that moment.